content/blog/2018/07/2018-07-10-jenkins-essentials-on-aws.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Jenkins Essentials flavor for AWS
+author: batmat
+tags:
+- jenkinsessentials
+- essentials
+---
+
+image:/images/logos/magician/256.png[Jenkins Essentials, role="right"]
+
+Jenkins Essentials is about providing a link:/blog/2018/04/06/jenkins-essentials/[distribution of Jenkins in less than five minutes and five clicks].
+One of the main ideas to make this a reality is that Jenkins will be autoconfigured with sane defaults for the environment it is running in.
+
+We are happy to report we recently merged the link:https://github.com/jenkins-infra/evergreen/pull/128[change that provides this feature for AWS].
+We use an _link:https://aws.amazon.com/cloudformation[AWS CloudFormation template]_ to provision a working version of _Jenkins Essentials_, automatically configured to:
+
+* dynamically provision EC2 agents, using the link:https://plugins.jenkins.io/ec2[EC2 plugin] ;
+* use the link:https://plugins.jenkins.io/artifact-manager-s3[Artifact Manager on S3] plugin, so that artifacts are not stored anymore on the master's FileSystem, but directly in an S3 bucket.
+
+I recorded a short demo video last week showing the basics of this:
+
+video::966OiFW2gow[youtube]
+
+While there are still many items to complete to provide a usable version for end-users, we are making steady progress towards it.
+
+---
+
+You can learn more about Jenkins Essentials from
+link:https://github.com/jenkins-infra/evergreen[GitHub repository], or join us
```

Comment:
GitHub repository is lacking article.

Reply:
I copied it from https://jenkins.io/blog/2018/06/26/jenkins-essentials-at-eclipsecon-france/, but I think you're right.